spec/operations/components/on_success_spec.rb

```diff
@@ -56,36 +62,11 @@
           ],
           errors: be_empty
         )
-      expect(after_commit).to have_received(:call).exactly(3).times
+      expect(after_commit).to have_received(:call).once
```

Comment:
what happened here? Why is it once and was exactly 3 times before?

Reply:
Before every callback was wrapped withing `after_commit`, not they are all wrapped at once